app/experimenter/nimbus-ui/src/components/SidebarActions/index.tsx

```diff
@@ -0,0 +1,46 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { RouteComponentProps } from "@reach/router";
+import React from "react";
+import { getExperiment_experimentBySlug } from "../../types/getExperiment";
+import { LinkNav } from "../LinkNav";
+import "./index.scss";
+import { ReactComponent as Trash } from "./trash.svg";
+
+type SidebarModifyExperimentProps = {
+  testid?: string;
+  experiment: getExperiment_experimentBySlug;
+} & RouteComponentProps;
+
+export const SidebarActions = ({
+  experiment,
+  testid = "SidebarActions",
```

Comment:
Don't think we ever override this prop, may as well move its value directly into the JSX